carp.client.core/src/commonTest/kotlin/dk/cachet/carp/client/domain/StudyRuntimeTest.kt

```diff
@@ -10,8 +11,7 @@ import kotlin.test.*
 class StudyRuntimeTest
 {
     @Test
-    fun initialize_matches_requested_runtime()
-    {
+    fun initialize_matches_requested_runtime() = runBlockingTest {
         // Create a deployment manager which contains a 'smartphone study'.
         val ( deploymentManager, deploymentStatus) = createStudyDeployment( createSmartphoneStudy() )
```

Comment:
While you are at at, maybe also rename these `deploymentManager` to `deploymentService`. I seemingly forgot to rename them in `StudyRuntimeTest`, or this was more merge-foo.

Reply:
Dont you have permissions to push changes to this PR? If you do, feel free to make does changes.